dotcom-rendering/src/web/components/InteractiveContentsBlockComponent.stories.tsx

```diff
@@ -3,28 +3,31 @@ import { NumberedList } from '../../../fixtures/generated/articles/NumberedList'
 import { enhanceInteractiveContentsElements } from '../../model/enhance-interactive-contents-elements';
 import { InteractiveContentsBlockComponent } from './InteractiveContentsBlockComponent';
 
-// @ts-expect-error: we know that NumberedList fixture has an interactive content block
-const interactiveContentsBlock: InteractiveContentsBlockElement =
-	enhanceInteractiveContentsElements(NumberedList.blocks)[0].elements.find(
-		(block) =>
-			block._type ===
-			'model.dotcomrendering.pageElements.InteractiveContentsBlockElement',
-	);
+const interactiveContentsBlock = enhanceInteractiveContentsElements(
+	NumberedList.blocks,
```

Comment:
I can see now we're getting an ESLint error: `Unsafe member access .blocks on an `any` value`. Is that okay?

Reply:
Just realised this was already happening!